lib/bulk/common.js

```diff
@@ -441,13 +444,519 @@ const BulkWriteError = function(error, result) {
 };
 util.inherits(BulkWriteError, MongoError);
 
+/**
+ * Add a single update document to the bulk operation
+ *
+ * @method
+ * @param {object} updateDocument update operations
+ * @param {class} bulk FindOperatorsOrdered or FindOperatorsUnordered
+ * @throws {MongoError}
+ * @return {OrderedBulkOperation or UnordedBulkOperation}
+ */
+const Update = function(updateDocument, bulk) {
+  // Perform upsert
+  var upsert = typeof bulk.s.currentOp.upsert === 'boolean' ? bulk.s.currentOp.upsert : false;
+
+  // Establish the update command
+  var document = {
+    q: bulk.s.currentOp.selector,
+    u: updateDocument,
+    multi: true,
+    upsert: upsert
+  };
+
+  // Clear out current Op
+  bulk.s.currentOp = null;
+  return bulk.addToOperationsList(bulk, UPDATE, document);
+};
+
+/**
+ * Add a single update one document to the bulk operation
+ *
+ * @method
+ * @param {object} updateDocument update operations
+ * @param {class} bulk either FindOperatorsOrdered or FindOperatorsUnordered
+ * @throws {MongoError}
+ * @return {OrderedBulkOperation or UnordedBulkOperation}
+ */
+const UpdateOne = function(updateDocument, bulk) {
+  // Perform upsert
+  var upsert = typeof bulk.s.currentOp.upsert === 'boolean' ? bulk.s.currentOp.upsert : false;
+
+  // Establish the update command
+  var document = {
+    q: bulk.s.currentOp.selector,
+    u: updateDocument,
+    multi: false,
+    upsert: upsert
+  };
+
+  // Clear out current Op
+  bulk.s.currentOp = null;
+  bulk.addToOperationsList(bulk, UPDATE, document);
```

Comment:
is there a reason we `return bulk.addToOperationsList` in `Update`, but we don't return the value in `UpdateOne`?